examples/hello_worlds.rb

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env ruby
+
+require 'vedeu'
```

Comment:
I've started adding `require 'bundler/setup'` to the top of each of the executable examples. This means that you can use `bundle exec appname` or `./appname` freely. (I tested this with the `editor_app.rb` and it appeared to work as expected. Could you update your pull request to include this line, and let me know if you get the same behaviour.

```
require 'bundler/setup'
require 'vedeu'

fail Vedeu::VERSION
```

Try that in this example and let me know the outcome. If all is rosy, please add the `require 'bundler/setup'` line to this pull request.